internal/gateway/proxy/proxy.go

```diff
@@ -147,8 +147,9 @@ type ControllerProxy struct {
 }
 
 func (cp *ControllerProxy) Publish(ctx context.Context, req *proxypb.PublishRequest) (*emptypb.Empty, error) {
-	if req.EventbusName == "" {
-		return nil, v2.NewHTTPResult(http.StatusBadRequest, "invalid eventbus name")
+	meta, _ := cp.ctrl.EventbusService().GetEventbus(ctx, req.EventbusId)
```

Comment:
why not check err directly

Reply:
if there has an error, the meta must be nil, when meta is nil, it will return